available_images.md

```diff
@@ -142,12 +142,23 @@ SageMaker Framework Graviton Containers (SM support only)
 
 NVIDIA Triton Inference Containers (SM support only)
 ============================
-**Note**:
+**Versions 23.12 and onwards**:
+1. Starting version 23.12, onwards, the sagemaker tritonserver is available in a different set of accounts than the previous ones. The new accounts are now the same as other DLCs listed on this page, making it easier to switch containers going forward. These accounts are listed in section '# Available Deep Learning Containers Images' above.
+2. Alternately, they can be obtained programmatically from the sagemaker python sdk as:
+```
```

Comment:
Code snippet a bit unclear - not sure what module needs to be imported, and customer will need to make sure they have the latest pysdk version in order to get the latest tritonserver image

Reply:
Added an import line to make it clear

Reply:
nit: actually, just thought of this, but might as well define all of the variables in this code block as well